<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="vertical_layout">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="interface_tab">
      <attribute name="title">
       <string>Interface</string>
      </attribute>
      <layout class="QVBoxLayout" name="vertical_layout_4">
       <item>
        <layout class="QFormLayout" name="form_layout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="Grid">
           <property name="text">
            <string>Grid Appearance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="grid_appearance_type_combo_box">
           <item>
            <property name="text">
             <string>Points</string>
            </property>
            <property name="icon">
             <iconset resource="cadnano.gui.ui.dialogs.dialogicons.qrc">
              <normaloff>:/part/grid_points</normaloff>:/part/grid_points</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lines</string>
            </property>
            <property name="icon">
             <iconset resource="cadnano.gui.ui.dialogs.dialogicons.qrc">
              <normaloff>:/part/grid_lines</normaloff>:/part/grid_lines</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="zoom_speed_label">
           <property name="text">
            <string>Mousewheel zoom speed:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="zoom_speed_slider">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="show_icon_label_text">
           <property name="text">
            <string>Show Icon Labels:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="show_icon_labels">
           <property name="text">
            <string>(needs restart)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="legacy_slice_view_label">
           <property name="text">
            <string>Slice View</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="legacy_slice_view_combo_box">
           <item>
            <property name="text">
             <string>Legacy Slice View</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Advanced Slice View</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="standardButtons">
          <set>QDialogButtonBox::RestoreDefaults</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="plugins_tab">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QVBoxLayout" name="vertical_layout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Plugins allow custom python code execution within cadnano.
See github.com/cadnano/plugins for examples.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="plugin_table_widget">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Path</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_plugin_button">
         <property name="text">
          <string>Add Plugin</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>zoom_speed_slider</tabstop>
 </tabstops>
 <resources>
  <include location="cadnano.gui.ui.dialogs.dialogicons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>addPlugin()</slot>
  <slot>handleButtonClick(QAbstractButton*)</slot>
  <slot>setHoneycombRows(int)</slot>
  <slot>setHoneycombCols(int)</slot>
  <slot>setHoneycombSteps(int)</slot>
  <slot>setSquareRows(int)</slot>
  <slot>setSquareCols(int)</slot>
  <slot>setSquareSteps(int)</slot>
  <slot>setAutoScaf(int)</slot>
  <slot>setStartupTool(int)</slot>
  <slot>setZoomSpeed(int)</slot>
 </slots>
</ui>
